packages/ui/src/components/w3m-text/index.ts

```diff
@@ -14,8 +14,6 @@ type Variant =
   | 'xsmall-normal'
   | 'xxsmall-bold'
 
-type Align = 'center' | 'left' | 'right'
-
```

Comment:
NB: Just to check all of this removal is related to the `all:inherit` pattern?

Reply:
Moreso to simplify text props, as this can be managed with css easily where needed.